paparazzi/paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PrepareResourcesTask.kt

```diff
@@ -79,27 +73,22 @@ abstract class PrepareResourcesTask : DefaultTask() {
     } else {
       mainPackage
     }
-    val projectDirectory = buildDirectory.get()
 
     out.bufferedWriter()
       .use {
         it.write(mainPackage)
         it.newLine()
-        it.write(projectDirectory.relativize(mergeResourcesOutput.get()))
+        it.write(mergeResourcesOutput.get())
         it.newLine()
         it.write(targetSdkVersion.get())
         it.newLine()
         // Use compileSdkVersion for system framework resources.
         it.write("platforms/android-${compileSdkVersion.get()}/")
         it.newLine()
-        it.write(projectDirectory.relativize(mergeAssetsOutput.get()))
+        it.write(mergeAssetsOutput.get())
         it.newLine()
         it.write(resourcePackageNames)
         it.newLine()
       }
   }
-
-  private fun Directory.relativize(child: Directory): String {
-    return asFile.toPath().relativize(child.asFile.toPath()).toFile().invariantSeparatorsPath
-  }
```

Comment:
Moved this function to the plugin to relativize the path input for mergeAssetsOutput and mergeResourcesOutput

Reply:
can we move this function as is to the plugin as a private extension method?